src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -114,7 +114,7 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
                     value={this.state.values[attrName].value}
                     onChange={this.updateValues.bind(this)}
                     data-attr-name={attrName}>
-                    {config.choices.map(this.renderChoice.bind(this))}
+                    {Object.entries(config.choices).map(this.renderChoice.bind(this))}
```

Comment:
Changing this from array to object seems like a BC break, so we cannot merge this into the 2.5.

Reply:
Sorry, can not agree with it. It is not a BC break. `choices` object (previously array) is not used anywhere else in the UI javascript except here. And the main reason why it was changed from array to object because previously it was containing only the list of choice values:
```
[`choice_1`, `choice_2`]
```
And now it has values and their labels:
```
{
    'choice_1': 'Choice 1',
    'choice_2': 'Choice 2'
}
```

So it is not a BC break. It is just a change. And it has no effect on previous versions. So it should be safe to merge it into `2.5`.
@alongosz @SylvainGuittard

Reply:
It is a BC break. We do not keep BC for our code but for third-party and we never know what was used in their code. This is a public configuration and can be used in any custom button for the AlloyEditor.

Reply:
> Sorry, can not agree with it. It is not a BC break. choices object (previously array) is not used anywhere

@SerheyDolgushev :upside_down_face: 
Handling OSS - a product - like eZ Platform - is entirely different ball game than handling projects. 

What you're saying is true for projects. But in our Product several elements are required to keep BC regardless of if we're using it or no. Exposed configuration is one of them. 

So I fully agree with what @dew326 said, moreover when it comes to BC in JS code (or code used by JS) he has the final word here :stuck_out_tongue_closed_eyes:

Reply:
Sorry, so what would be your suggestion? To add a new public configuration, which would contain choices translations (basically it will be the same as `choices` currently is in this PR)?

Reply:
Probably this is the only one choice here to keep the BC:
- We add a new key in the configuration at the same level as `choices` with `choicesLabel`.
- The `choicesLabel` should be an object with pairs value and label
```js
{
    value1: 'Label1',
    value2: 'Label2',
    value3: 'Label3',
}
```
- And now you can render the label in `renderChoice()` method by `choicesLabel[choice]`

WDYT @alongosz ?

Reply:
+1

Reply:
@dew326 It was done. Please let me know if there is a better way to access `choicesLabel ` in `EzBtnCustomTagUpdate.renderChoice` (https://github.com/ezsystems/ezplatform-admin-ui/pull/1001/commits/dbff697892079bf2fde8f69b0bd9b9e7a8d5f73e#diff-e1998a185854472882a7777bfd69d83cR109)